official-build.props

```diff
@@ -0,0 +1,3 @@
+DISABLE_AUTOTAGGING=true
```

Comment:
Check with Tony but I don't think the `DISABLE_AUTOTAGGING=true` line is required.

```suggestion
```

Reply:
@tonymcveigh Is this setting required anymore as per Dermots comment?

Reply:
No, that's an old setting. No longer required.